tests/e2e/testfiles/values.yaml

```diff
@@ -1420,7 +1420,9 @@
     # Example:
     #   ./hello_world.sh
     #   cert-csi test vio --sc <storage class> --chainNumber 2 --chainLength 2
-    run: ./cert-csi test vio --sc powermax --chainNumber 2 --chainLength 2
+    run: 
+      - ./cert-csi test vio --sc powermax --chainNumber 2 --chainLength 2
```

Comment:
Isn't this PR is for csi-powerscale deployments? If then why we are running on powermax sc? I understand sc content may be correct, but I think we should be consistent with naming convention too, to avoid confusion.

Reply:
Forgot to mention this in the description, a previous PR introduced a bug into the pmax e2e tests, I'm fixing that here.